src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -1016,30 +1008,38 @@ final case class EServicesApiServiceImpl(
       interfaceFile  <- fileManager
         .get(ApplicationConfiguration.eServiceDocumentsContainer)(interface.path)
         .map(_.toByteArray)
-      _              <- FileManagerUtils
-        .verify(interfaceFile, interface.name, eService, isInterface = true)
-        .toFuture
       docFiles       <- descriptor.docs.traverse(doc =>
         fileManager
           .get(ApplicationConfiguration.eServiceDocumentsContainer)(doc.path)
           .map(filestream => (filestream.toByteArray, doc.name))
       )
       config     = extractConfig(eService, descriptor)
-      folderName = s"${eService.name}_V${}${descriptor.version}"
+      folderName = s"${eService.id}_${descriptor.id}"
+      zipName    = s"$folderName.zip"
```

Comment:
not critical, but it could help
```suggestion
      zipName    = s"$organizationId/$folderName.zip"
```